src/mongodb/config/language/gu.ts

```
@@ -0,0 +1,6 @@
+var gu_config = {
+    vowel: ["ા", "િ", "ી", "ુ", "ૂ", "ૃ", "ે", "ૈ", "ો", "ૌ", "ં", "ઃ"],
+    language_code: "gu"
+};
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)